cmd/clusterctl/examples/aws/aws_manager_image_patch.yaml.template

```diff
@@ -8,4 +8,5 @@ spec:
     spec:
       containers:
       - image: ${MANAGER_IMAGE}
+        imagePullPolicy: IfNotPresent
```

Comment:
I believe this is already set here https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/config/manager/manager.yaml#L57

Reply:
@vincepri it's set to Always there. Do you want me to change it to IfNotPresent in the other location?

Reply:
That would break the development experience, is there a reason why `Always` isn't suitable here?

Reply:
Because during integration testing there's no repository available.

Reply:
@vincepri maybe we should leverage a kustomize patch here to set it to `Always` for dev workflows. `IfNotPresent` makes a lot more sense for end users.

Reply:
Or just change your dev workflow to use kind :wink: